Example/Draft/ViewController.swift

```diff
@@ -66,7 +66,7 @@ struct GetJSONQuote: JSONDraft {
 
 // Codable
 
-struct GetDecodableQuote: DecodableDraft {
+struct GetDecodableQuote: DecodableDraft {    
```

Comment:
enable Remove trailing whitespace option in Xcode please

Reply:
hmm it is enabled...

Reply:
Weird ok.